modules/plugin/jdbc/jdbc-postgis/src/test/java/org/geotools/data/postgis/PostGISUuidOnlineTest.java

```diff
@@ -5,8 +5,21 @@
 
 package org.geotools.data.postgis;
 
+import org.geotools.data.DataUtilities;
```

Comment:
Please update file header

Reply:
@jodygarnett this file has had the wrong header since before we switched to git, I don't think it's @petersmythe job to fix it. The QA checks only validate sources in ``src/main``, skipping the ones in ``src/test``. Try to enable it for test sources and you'll find out why (the hard way). If you feel it's important to fix it, do as I did, take your spare time and go over all the sources, make a pull request that enables the checks on tests and fixes all the missing/incorrect headers.

Reply:
fair enough, I just noticed because the file was modified.